charts/opentelemetry-operator/release-checklist.md

```diff
@@ -0,0 +1,34 @@
+# Prerequisite
+
+- [ ] Make sure you have installed Kuttl which will be used to do the smoke tests later. See [Kuttl website](https://kuttl.dev/docs/)
+  for installation information.
+- [ ] Make sure you have cloned the [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator) in your workspace
```

Comment:
Why this step is needed?

Reply:
It's because we want the maintainers to run [KUTTL smoke tests](https://github.com/open-telemetry/opentelemetry-operator/blob/main/kuttl-test.yaml) from the Operator repo against the Operator installed through Helm chart.